### PR TITLE
Pass through props to wordcloud element

### PR DESCRIPTION
**Pull request by davidjb**
This passes through all other props through to the underlying element, allowing props like ``className``, ``style``, and anything else HTML or React-related to be passed through.

----

From #50:

> My main use cases were:
>
> * Being able to add arbitrary accessibility props to the underlying div element (e.g. https://reactjs.org/docs/accessibility.html), so aria-label where necessary and aria-hidden when I wanted to exclude this and provide another more accessible-friendly option, and
> * Being able to use `className` and `style` to add styles to responsively target the word cloud from CSS media queries, adjusting the size according to device properties, primarily being width-based in my grid (e.g. specifically sized word clouds at different breakpoints; taller wordclouds on mobile and wider/medium height on desktop)
>
> On the latter, I suppose you could add a wrapping div but it adds extra extra markup (and a requirement to have a custom component vs just a className).  On the former, you could aria-hidden a higher level of element a screen reader would ignore the children, but the wrong element would be labelled applying aria-label (or similar a11y props) to the parent.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ function ReactWordCloud({
 	options,
 	size: initialSize,
 	words,
+	...passThroughProps
```
Comment (chrisrzhou) [inline src/index.js]: nit: rename to `rest` and same in line 67.

You might need to pull/rebase because I made changes a few minutes ago.

Comment (davidjb) [reply src/index.js]: Ah, yep sure, will update and rebase.  That's pushing now.

**Comment by chrisrzhou** [comment]
I think the remote `master` branch is removed, so it might be best to have a clean pull from `main` or refork and point the MR to that.  Your changes/diff is small so that might be fast enough to get things going smoothly 😅

**Comment by davidjb** [comment]
Should be good now @chrisrzhou -- rebased on `main`.  The only thing that was fiddly was the tabs to spaces change -- otherwise it rebased cleanly.

**Comment by chrisrzhou** [comment]
@davidjb, github seems to be complaining about conflicting files in `src/files`.  For reference, the up-to-date `main` branch has this code that I don't see in the diff view: https://github.com/chrisrzhou/react-wordcloud/blob/main/src/index.js#L45

Github's been pretty slow tonight, so I'm not sure if something's up with that, but let's sanity check the link above in your forked repo.

**Comment by chrisrzhou** [comment]
Anyway, this merge conflict is fairly easy to resolve (used the UI lol).  Just resolved it, so approving/merging it!  For your other MR, let's see if our `main` branches are up-to-date.

**Review by chrisrzhou — APPROVED** [review]

**Pull request merged**